cmake install is broken

Prevented 0.18.3 release:

https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=19541&view=logs&j=5435e0ac-25e5-5426-50be-61b0d0ea8d34&t=3e1ef318-4174-5329-41dd-e9ef66c4c6d0

```
exec ./tests/test_install_build.sh
========================== Starting Command Output ===========================
/bin/bash --noprofile --norc /mnt/build/_temp/4674c9ed-f997-4797-8a36-637afdbd163d.sh
+ mkdir build_against_install
+ cd build_against_install
++ which clang-8
+ CC=/usr/bin/clang-8
++ which clang++-8
+ CXX=/usr/bin/clang++-8
+ cmake -GNinja ../samples/apps/logging/
-- The C compiler identification is Clang 8.0.1
-- The CXX compiler identification is Clang 8.0.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang-8 - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++-8 - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:7 (find_package):
  By not providing "Findccf.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "ccf", but
  CMake did not find one.

  Could not find a package configuration file provided by "ccf" with any of
  the following names:

    ccfConfig.cmake
    ccf-config.cmake

  Add the installation prefix of "ccf" to CMAKE_PREFIX_PATH or set "ccf_DIR"
  to a directory containing one of the above files.  If "ccf" provides a
  separate development package or SDK, be sure it has been installed.
```

--- a/cmake/version.cmake
+++ b/cmake/version.cmake
@@ -9,7 +9,7 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
   find_package(Git)
 
   execute_process(
-    COMMAND ${GIT_EXECUTABLE} describe --tags
+    COMMAND "bash" "-c" "${GIT_EXECUTABLE} describe --tags | grep ccf-"
     OUTPUT_VARIABLE "CCF_VERSION"
     OUTPUT_STRIP_TRAILING_WHITESPACE
     RESULT_VARIABLE RETURN_CODE
@@ -17,7 +17,7 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
   if(NOT RETURN_CODE STREQUAL "0")
     message(
       FATAL_ERROR
-        "Git repository does not appear to contain any tag (the repository should be cloned with sufficient depth to access the latest \"ccf-*\" tag)"
+        "Git repository does not appear to contain any tag starting with ccf- (the repository should be cloned with sufficient depth to access the latest \"ccf-*\" tag)"
     )
   endif()
   execute_process(

